tests/issues/x86-prefixes.yaml

```
@@ -0,0 +1,217 @@
+test_cases:
+  # Test segment override priority
+  -
+    input:
+      name: "x86-16: rightmost segment override should take priority"
+      bytes: [ 0x26, 0x65, 0x64, 0x3E, 0x65, 0x2E, 0x00, 0x00 ]
+      arch: "CS_ARCH_X86"
+      options: [ CS_MODE_16 ]
```

```suggestion
      options: [ CS_MODE_16, CS_OPT_DETAIL ]
```

Otherwise details are not decoded.